frontend/src/components/Projects/MyProject.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Link } from "react-router-dom";
 import ProjectsApi from "./../../api/ProjectsApi";
-import "./MyProjects.css";
```

Comment:
Why removing this? Don't we need this css file?  if don’t then i guess that file should be removed